src/Analysis/Ast/Impl/Modules/PythonModule.cs

```diff
@@ -148,9 +149,26 @@ internal PythonModule(ModuleCreationOptions creationOptions, IServiceContainer s
         #endregion
 
         #region IMemberContainer
-        public virtual IMember GetMember(string name) => GlobalScope.Variables[name]?.Value;
 
-        public virtual IEnumerable<string> GetMemberNames() => GlobalScope.GetExportableVariableNames();
+        public virtual IMember GetMember(string name) {
```

Comment:
This modifies all accesses, not just the completion for imports. Is this what you expect? Not sure if we had removed this behavior or modified at some point in the past or not... Vaguely remember something like this previously and having to be careful to present things in the right place.

Reply:
Yes, if it is a legal member Python can find, it should be exposed. I looked into past history and found that handling of submodules was added for import statement completion specifically. This code is basically borrowed from there.